src/Utils.vala

```diff
@@ -39,6 +39,33 @@ namespace Utils {
         }
     }
 
+    public uint get_file_age (GLib.File file) {
+        FileInfo info;
+        try {
+	        info = file.query_info (FileAttribute.TIME_MODIFIED, FileQueryInfoFlags.NONE);
+        } catch (Error e) {
+            warning ("Error while getting file age: %s", e.message);
+            return uint.MAX;
+        }
+
+	    if (info == null) {
+            return uint.MAX;
+        }
+
+	    uint64 mtime = info.get_attribute_uint64 (FileAttribute.TIME_MODIFIED);
+	    uint64 now = (uint64) get_real_time () / 1000000;
```

Comment:
Wouldn't https://valadoc.org/glib-2.0/time_t.html make more sense than this very precise time?

Reply:
`FileAttribute.TIME_MODIFIED` is a `uint64` and I personally prefer getting the mtime this way as we can use `FileAttribute` and proper GLib exception handling. If we wanted the mtime as time_t, I think we'd have to use something like https://valadoc.org/glib-2.0/GLib.Stat.html which isn't as clean IMO.

Let me know what you think, or if you had a different idea.

Reply:
I'm more talking about the `now` value as you don't need the precision of `get_real_time ()`